matchzoo/models/anmm_model.py

```diff
@@ -0,0 +1,70 @@
+"""An implementation of aNMM Model."""
+
+
+import logging
+
+import keras
+from keras.activations import softmax
+from keras.initializers import RandomUniform
+
+from matchzoo import engine
+
+logger = logging.getLogger(__name__)
+
+
+class ANMMModel(engine.BaseModel):
+    """
+    ANMM Model.
+
+    Examples:
+        >>> model = ANMMModel()
```

Comment:
Please add the anmm-specific parameter explicitly configuration here, since the `Examples` is to show how to use `anmm`.  I have just updated existing models. You can check it.

Reply:
I think it's ok to ignore the parameter setting for now. All models parameters need to be documented somehow, and I will generate part of those docs automatically later.

Reply:
That's great to automatically generate the docs.